src/Nest/Indices/Analyze/AnalyzeRequest.cs

```diff
@@ -42,29 +46,33 @@ public AnalyzeRequest(IndexName indices, string textToAnalyze)
 			this.Text = new[] { textToAnalyze };
 		}
 
-		///<summary>The name of the analyzer to use</summary>
+		/// <inheritdoc />
 		public string Analyzer { get; set; }
 
-		///<summary>A collection of character filters to use for the analysis</summary>
+		/// <inheritdoc />
+		public string Normalizer { get; set; }
+
+	    /// <inheritdoc />
 		public string[] CharFilter { get; set; }
 
-		///<summary>A collection of filters to use for the analysis</summary>
-		public string[] Filter { get; set; }
+	    /// <inheritdoc />
+	    public string[] Filter { get; set; }
```

Comment:
spaces instead of tabs

Reply:
https://i.imgur.com/3bN0wfg.png?fb

:)